contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java

```diff
@@ -260,6 +292,17 @@ protected Statement setStatementParameters(PreparedStatement updateCommand, Obje
     return boundStmnt;
   }
 
+  @Override
+  public void processTuple(Object tuple)
+  {
+    try {
+      super.processTuple(tuple);
+      recordsProcessed++;
+    } catch (RuntimeException e) {
+      //super.error.emit(tuple);
```

Comment:
In #215 you have added error port to, AbstractTransactionableStoreOutputOperator operator which is parent to this operator. I will update my pull request once I rebase with latest changes.